test/Container/RegistryApiSpec.hs

```diff
@@ -68,6 +68,16 @@ registryApiSpec :: Spec
 registryApiSpec =
   describe "Container Registry API" $
     describe "Public Registry APIs" $ do
+      describe "Microsoft registry" $ do
```

Comment:
When I run this on my laptop, I see this:

```
make test-cabal ARGS="Container Registry API"
```

```
 test/Test/Effect.hs:125:33: 
  1) Container.RegistryApi, Container Registry API, Public Registry APIs, Microsoft registry, should get manifest with digest
       expected: RepoDigest "sha256:bd36eac352dbb2d27a63f4714b9063fc8c7e9c615e430006dbeb7e3448cd5780"
        but got: RepoDigest "sha256:d24e8ccc34df12a86f907d27d85bfa732f30c771bfd9bd7e4ebf4ba66f18b241"
```

Is this because something changed on the registry or because I'm downloading something different because I'm on macOS or ???

Reply:
No os or arch shouldn't matter - there was an actual update to the tag (they overwrote the image@tag) - updated with more stale example now

Reply:
:+1: